### DISCUSSION TITLE
Fundamentos do HTML e CSS

### DISCUSSION THREAD

**Discussion by jefersonWudeson** [balta-io/forum#526] [Tira Dúvidas] [answered]
Olá comecei a pouco tempo fazendo um curso no balta só que eu estou tendo uma dificuldade com a ingressão de uma imagem dentro da borda ja tentei algunhas vezes mudando o nome da imagem mudando a imagem de lugar e mudando de imagem e nao conseguir resolver
![css2](https://github.com/balta-io/forum/assets/164440889/e290a2d8-1288-45c2-a048-d2f4e844007f)
tentei colocar essa imagem ai so que nao deu certo 
![1Capturar](https://github.com/balta-io/forum/assets/164440889/ea7109f5-4eb5-4e4a-87fb-8c6d16502775)
![2Capturar](https://github.com/balta-io/forum/assets/164440889/35334f34-f81e-4087-95cc-a3cc56b9a145)
![3Capturar](https://github.com/balta-io/forum/assets/164440889/36dc3e1b-c16f-48bc-b690-93318bc7583c)
o site fica assim e nao aparece a foto dentro da borda
![4Capturar](https://github.com/balta-io/forum/assets/164440889/ca26aa70-e3de-49b1-86d8-e5a16b6ce3f8)
o codigo esta igual e mesmo assim nao deu certo
![5Capturar](https://github.com/balta-io/forum/assets/164440889/b2804db1-d7de-44e3-bac5-c974a27f3918)
e era para ele ficar assim so que nao aparece a foto

**Comment by henriquegiaretta** [comment, marked as answer]
Bom dia, Jeferson, tudo bem?

Eu vejo dois pontos que possam estar causando o problema. São eles:

1) Se você está utilizando **Live Server**, extensão que eu mostro no curso, pode ser que ele não esteja conseguindo renderizar a imagem. Vi em alguns poucos casos que ele não é performático para uso de muitas imagens. Tente abrir o seu arquivo **.html** diretamente no browser, sem o uso do Live Server.

2) Eu vejo que você está utilizando a contra barra (\) ao invés da barra (/) no caminho da imagem. É possível que não leia corretamente o arquivo devido a essa mudança da barra.

Outro detalhe que você pode fazer para testes é: tente usar a mesma url em uma tag **<img>** (tem aula no curso sobre o uso desta tag) e veja se você será capaz de ver a imagem.

Espero ter ajudado. Qualquer duvida é só mandar por aqui.

Grande abraço.
